### PR TITLE
Resolve 5# Created static homepage

### PR DESCRIPTION
**Pull request by abalanag**
- Created static homepage
- Added a carousel on the first page

### CODE REVIEW

File: src/main/resources/footerScripts.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```
Comment (danutchindris) [inline src/main/resources/footerScripts.html]: You should place these fragments in `common.html`.

File: src/main/resources/static/homepage.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<!doctype html>
```
Comment (danutchindris) [inline src/main/resources/static/homepage.html]: Not sure what you want to do with this, but the `homepage.html` Thymeleaf template should reside in `resources/templates`, not in the `static` folder.
That's why you won't need the `spring.resources.static-locations=classpath:/static/` property in your `application.properties` file.

File: src/main/resources/static/homepage.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <title>Airports - Homepage</title>
+</head>
+<body>
+<br><br><br><br><br>
+<center>
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+        <style>
```
Comment (danutchindris) [inline src/main/resources/static/homepage.html]: I think it's bad practice to pollute the markup with CSS code when you can place it in a dedicated file.

File: src/main/resources/static/homepage.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    </div>
+</center>
+
+<!-- Optional JavaScript -->
+<!-- jQuery first, then Popper.js, then Bootstrap JS -->
+<link href="#" th:replace="footerScripts :: bootstrap-js"/>
+<link href="#" th:replace="footerScripts2 :: bootstrap-js"/>
+<link href="#" th:replace="footerScripts1 :: bootstrap-js"/>
+</body>
+</html>
```
Comment (danutchindris) [inline src/main/resources/static/homepage.html]: Empty line here please.

**Review by danutchindris — CHANGES_REQUESTED** [review]
Please see the review comments and perform the necessary changes.
Thanks!

File: src/main/resources/footerScripts.html (hunk 1 above)
Comment (abalanag) [reply src/main/resources/footerScripts.html]: fragments added to common.html

File: src/main/resources/static/homepage.html (hunk 2 above)
Comment (abalanag) [reply src/main/resources/static/homepage.html]: deleted `spring.resources.static-locations=classpath:/static/` from `application.properties` and created controller in order to access homepage

File: src/main/resources/static/homepage.html (hunk 3 above)
Comment (abalanag) [reply src/main/resources/static/homepage.html]: moved CSS code into a CSS file

File: src/main/resources/static/homepage.html (hunk 4 above)
Comment (abalanag) [reply src/main/resources/static/homepage.html]: Empty line added

**Review by danutchindris — APPROVED** [review]
OK

**Pull request merged**